### PR TITLE
adds draft of support program criteria

### PR DESCRIPTION
**Pull request by nellshamrell**
Moving this from our shared Google doc to the GitHub repo.

### CODE REVIEW

File: docs/project-support-program.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Support Models
+
+The SIG supports the following modes of support
+
+* **Request funding** - projects may seek financial funding (hard-cost), where dedicated resources can make a difference in their development and sustainability. These projects often include software solutions addressing critical challenges in the open source ecosystem
+
+* **Request resource support** - projects that request resource (soft-cost) support, such as access to technical expertise or collaboration opportunities. These encompass a wide spectrum, from educational platforms to initiatives promoting open-source adoption of memory safeness.
+
+A project can submit its support request through the SIG repository issues (TODO: add issue template) and project members are welcome to join the SIG bi-weekly calls to present their work and support request.
```
Comment (balteravishay) [inline docs/project-support-program.md:15]: added issue to track the TODO: #14

**Comment by balteravishay** [comment]
discussed in the last SIG call: 
should this topic be managed at a SIG level, or a WG or even foundation level? it wouldn't change that much if it was a Security Tooling or BEST WGs project, for instance?

**Comment by SecurityCRob** [comment]
From a TAC/Foundation perspective, we're working out a funding model and processes to potentially routing resources to petitioners seeking assistance.  Details are all still "TBD", and as they solidify there will be a clearly documented process, criteria for consideration, and a business case template to help capture key data so that the GB can make a funding decision.  I think the words here are fine for now, but will need updated as that process solidifies.  It may be a good idea to start documented desirable criteria for MemSafe efforts, since we absolutely know this is a key concern within the industry and with government bodies.  I expect this little group will be getting a lot of attention in the coming year, and planning up front here will make our duties simpler as things get moving.

**Comment by nellshamrell** [comment]
Closing in favor of #16

**Pull request closed without merging**